app/models/concerns/date_helpers.rb

```diff
@@ -0,0 +1,14 @@
+module DateHelpers
+
+  private
+
+  def valid_date?(date)
+     Date.parse(date.to_s) and return true
```

Comment:
@christophermlne your suggestion is close to what was there initially. If `Date.parse` succeeds, its return value is a date object, which is truthy. The statement you propose then becomes `return true if true`, which is clearly redundant. This can then be shortened to `return Date.parse` or, since Ruby automatically returns the result of the last executed statement, simply `Date.parse`, which is what we had in this module to begin with. Ultimately, `true and return true` is not substantially different or less redundant than `return true if true`.

@razorcd will correct me if I'm wrong, but his desire to rewrite this arose from the fact that `valid_date?` is a predicate method, and it would be good for it to explicitly return `true` or `false`.

It is possible to coerce any value to a boolean by prepending it with `!!`, e.g. `!!Date.parse('01-01-2015') # true` - would that be an improvement?